.appveyor/install.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+export PATH="/mingw64/bin:${PATH}"
+export PKG_CONFIG_PATH="/mingw64/lib/pkgconfig:${PKG_CONFIG_PATH}"
+export GOROOT=/mingw64/lib/go
+export GOPATH=/go
+export CGO_ENABLED=1
+
+pacman --noconfirm -S \
+    mingw64/mingw-w64-x86_64-go \
+    mingw64/mingw-w64-x86_64-libusb
+
+go version
+go get github.com/google/gousb/...
+go test github.com/google/gousb/...
+go build -o gousb-win64.exe github.com/google/gousb
```

Comment:
does that make sense? "usb" does is not an executable, but a package. It probably should not be an "exe". Can we skip that entirely?

My understanding is that if you need to use gousb in your Windows Go project, you'll need to set up the whole build environment anyway. And in gousb itself, there's nothing really useful other than a few test programs.